AAI/README.md

```diff
@@ -19,160 +19,98 @@ permalink: aai-introduction
-then in some downstream application that protects data.*
-
-**Data Owners:**
-
-Data owners are not required to implement or operate an Identity Provider
-(though they may choose to do so) or an Broker.
-
-Data Owners may choose to operate a Claim Clearinghouse server configured
-to consume access_tokens from an upstream Broker and then hand out JWT
-claims to relying parties and other Claim Clearinghouses.
-
-Some data owners will own the whole "chain" providing all of the different kinds
-of brokers and will also operate Claim Clearinghouses. For instance, NIH is a
-data owner and might provide Cloud Buckets and operate an IDP and Broker to
-utilize ERACommons and other identity resources.
-
-A Data Owner should be able to, based on an Identity from an Identity Provider,
-express some sort of [permissions](#ga4gh-jwt-format) via the Claim Clearinghouse
-GA4GH claims. It is the responsibility of the Data Owner to provide these
-permissions to their Claim Clearinghouse to be expressed claims within a standard
-/userinfo process for downstream use.
-
-It is possible that the IdPs might have special claims. The Claim Clearinghouse
-being operated by the Data Owner should be "looking" for those claims and
-incorporating them, if desired, into the claims that it eventually sends to the
-user.
-
-A data owner is expected to maintain the [operational
-security](https://github.com/ga4gh/data-security) of their Claim Clearinghouse
-server and hold it to the GA4GH spec for [operational
-security](https://github.com/ga4gh/data-security). It is also acceptable to
-align the security to a known and accepted framework such as NIST-800-53,
-ISO-27001/ISO-27002.
-
-### Future topics to explore
-
-<https://openid.net/specs/openid-connect-federation-1_0.html> - OIDC federation
-
-Register the Claim - According to RFC 7519 (JSON Web Token) section 4.2
-<https://tools.ietf.org/html/rfc7519#section-4.2> claim names should be
-registered by IANA in its "JSON Web Token Claims" registry at
-<https://www.iana.org/assignments/jwt/jwt.xml> . Register GA4GH.
+level that they cannot be ignored. The use of a Broker and Clearinghouse
+enables attaching information to the usual OIDC flow so that Google and other
+prominent identity providers can be used with customized claims and scopes.
+
+Here is a diagram of a single broker. This is one possible way to use this spec.
+
+@startuml
+skinparam componentStyle rectangle
+left to right direction
+
+component "<b>Visa Issuer 1</b>\nservice" as Issuer1
+component "<b>Visa Issuer 2</b>\nservice" as Issuer2
+component "<b>Visa Issuer N</b>\nservice" as IssuerN
+
+component "<b>Broker</b>\nservice" as Broker #FAFAD2
+component "<b>Passport Clearinghouse</b>\nservice" as ClearingHouse #9E7BB5
+
+Issuer1 <-- Broker : Fetch Visas
+Issuer2 <-- Broker : Fetch Visas
+IssuerN <-- Broker : Fetch Visas
+Broker <-- ClearingHouse : Request User Visas w/ Access Token
+IssuerN <-- ClearingHouse : Request Public Key w/ JWKS
```

Comment:
```suggestion
```

Reply:
I think this arrow can be removed without confusion. 

While we do require clearinghouses to validate signatures, we do not want to clearinghouses to just follow JKU claims without allowlisting. My worry is that the diagram implies the [insecure usage of JKU](https://www.rfc-editor.org/rfc/rfc8725.html#name-do-not-trust-received-claim). Leaving the arrow out does not imply a lack of validation, it's just a detail that's not needed for this introduction.